src/status_im/contexts/wallet/common/activity_tab/view.cljs

```diff
@@ -88,20 +90,62 @@
+  [rn/text
+   {:style    {:text-decoration-line :underline}
+    :on-press on-press} text])
+
 (defn view
   []
-  (let [theme         (quo.theme/use-theme)
-        activity-list (rf/sub [:wallet/activities-for-current-viewing-account])]
-    (if (empty? activity-list)
-      [empty-tab/view
-       {:title       (i18n/label :t/no-activity)
-        :description (i18n/label :t/empty-tab-description)
-        :image       (resources/get-themed-image :no-activity theme)}]
-      [rn/section-list
-       {:sections                       activity-list
-        :sticky-section-headers-enabled false
-        :style                          {:flex               1
-                                         :padding-horizontal 8}
-        :content-container-style        {:padding-bottom jump-to.constants/floating-shell-button-height}
-        :render-fn                      activity-item
-        :render-section-header-fn       (fn [{:keys [title]}] [quo/divider-date title])}])))
+  (let [theme                    (quo.theme/use-theme)
+        address                  (rf/sub [:wallet/current-viewing-account-address])
+        activity-list            (rf/sub [:wallet/activities-for-current-viewing-account])
+        open-eth-chain-explorer  (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/mainnet-network-name}])
+                                  [address])
+        open-oeth-chain-explorer (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/optimism-network-name}])
+                                  [address])
+        open-arb-chain-explorer  (rn/use-callback
+                                  #(rf/dispatch [:wallet/navigate-to-chain-explorer
+                                                 {:address address
+                                                  :network constants/arbitrum-network-name}])
+                                  [address])]
+    [:<>
+     [quo/information-box
+      {:type      :informative
+       :icon      :i/info
+       :closable? false
+       :style     {:margin-horizontal 20 :margin-vertical 8}}
+      [:<>
+       [rn/text (str (i18n/label :t/wallet-activity-beta-message) " ")]
+       [pressable-text
+        {:on-press open-eth-chain-explorer
+         :text     (i18n/label :t/etherscan)}]
+       [rn/text ", "]
+       [pressable-text
+        {:on-press open-oeth-chain-explorer
+         :text     (i18n/label :t/op-explorer)}]
+       [rn/text (str ", " (string/lower-case (i18n/label :t/or)) " ")]
+       [pressable-text
+        {:on-press open-arb-chain-explorer
+         :text     (i18n/label :t/arbiscan)}]
+       [rn/text "."]]]
```

Comment:
This is just a vague suggestion:

I'm not 100% sure, but the `rn/text` component can be nested, so maybe we could make this less verbose if we change from:

```clojure
[:<>
 [rn/text "A"]
 [rn/text "B"]
 [pressable-text "C"]
 [rn/text "D"]]
```
to:
```clojure
[rn/text
 "A"
 "B"
 [pressable-text "C"]
 "D"]
```

Reply:
It's right, actually. I used `rn/text` (instead of `quo/text`) to inherit the text style of the component. We can remove the `rn/text` for the text every time. We don't even need to wrap `rn/text`, just a fragment `:<>` would suffice as the component has the `rn/text`.

```clojure
[:<>
 "A"
 "B"
 [pressable-text "C"]
 "D"]
```